cli/configssh.go

```diff
@@ -18,21 +27,82 @@ import (
 	"github.com/coder/coder/codersdk"
 )
 
-const sshStartToken = "# ------------START-CODER-----------"
-const sshStartMessage = `# This was generated by "coder config-ssh".
+const (
+	sshDefaultConfigFileName      = "~/.ssh/config"
+	sshDefaultCoderConfigFileName = "~/.ssh/coder"
+	sshCoderConfigHeader          = "# This file is managed by coder. DO NOT EDIT."
+	sshCoderConfigDocsHeader      = `
 #
-# To remove this blob, run:
+# You should not hand-edit this file, all changes will be lost upon workspace
+# creation, deletion or when running "coder config-ssh".`
```

Comment:
I really would love to see this feature. (i.e. running `coder create` will automatically update the config). But I suggest we rewrite this comment to not include it, open up a ticket for this functionality, and leave it for a follow-up PR.

Reply:
TODO: rewrite comment